docs/titan-reference/button-reference.md

```diff
@@ -0,0 +1,485 @@
+
+ --- 
+## \<Right\> 
+ (D9 / ST only) Right mouse button when trackball in mouse mode.
+
+ --- 
+## \<Select If\> 
+ Selects all fixtures which have intensity above zero. See [Select If](../controlling-fixtures.md/#selecting-fixtures-which-are-in-a-playback).
+
+ --- 
+## \<Set\> 
+ This button has become the \<Time\> button.
+
+ --- 
+## \<Shape\> 
+ Selects the Shape/FX menu to run shapes, key frame shapes or pixel map effects. See [Shapes and Effects](../effects.md).
+
+ --- 
+## \<Size/Position\> 
+ Moves the currently selected workspace window around the different size and position options. If \<Avo\> held also, moves the window to the other screen.
+
+ --- 
+## \<Snap\> 
+ Enables Snap mode which makes \<Prev Step\> / \<Next Step\> buttons snap through cues without using the programmed fade times. See [Running a cue list](../cue-lists/cue-list-playback.md/#running-a-cue-list).
+
+ --- 
+## \<Special\> 
+ Selects the Special attribute group so the wheels control fixture-specific functions like modes and macros. Also used for creating masks. See [Changing Attributes using the Wheels](../controlling-fixtures/changing-fixture-attributes.md/#changing-attributes-using-the-wheels).
+
+ --- 
+## \<Stop\> 
+ Stops/freezes the current cue fades. See [Running a cue list](../cue-lists/cue-list-playback.md/#running-a-cue-list).
+
+ --- 
+## \<Through\> 
+ Used when selecting ranges of items. See [Selecting fixtures and dimmers by number](../controlling-fixtures.md/#selecting-fixtures-and-dimmers-by-number-channel).
+
+ Example: \<Fixture\> 1 \<Through\> 5 \<not\> 3 selects fixtures 1, 2, 4, 5
+
+ --- 
+## \<Time\> 
+ Show the Times menu which is used to set cue timing. See [Fade times](../cues/cue-timing.md/#fade-times-and-fixture-overlap).
+
+ --- 
+## \<Undo\> 
+ Undo the last action (an Undo list is shown in the system prompt). See [Undo/Redo](../titan-basics/other-parts-of-the-touch-screen.md/#undoredo).
+
+ --- 
+## \<Unfold\> 
+ followed by playback **Select**, splits out a cue list or chase onto the playback faders so each cue/step can be edited individually. See [Editing a chase using unfold](../chases/editing-a-chase.md/#editing-a-chase-using-unfold).
+
+ --- 
+## \<Update\> 
+ Updates the current cue by merging the contents of the programmer. See [Updating Stored Values and Palettes used in a Cue](../cues/editing-cues.md/#updating-stored-values-and-palettes-used-in-a-cue).
+
+ --- 
+## \<View\> 
+(D9, Titan Go, Pearl Expert, Tiger Touch 1) \
+ Followed by **Select** button to show details about item, or double press to open workspace window. See [Palettes](../palettes/editing-palettes.md/#viewing-and-editing-the-content-of-palettes) and [Cues](../cues/editing-cues.md/#playback-and-cue-view), or [Opening Workspace Windows](../titan-basics.md/workspace-windows/#opening-and-positioning-workspace-windows).
```

Comment:
Also followed by various other buttons to open specific workspace windows:
`<View><Patch>` - Patch view
`<View><Playback>` - Playbacks view
`<View><Fixture>` - Fixtures window
`<View><Group>` - Groups window
`<View><Macros>` - Macros window
`<View><Options>` - Attribute editor
`<View><Off>` - Active playbacks window
`<View><Release>` - Active playbacks window
`<View><Connect>` - Playback view for connected playback
`<View><Intensity>` - Intensity View
`<View><Position>` - Position Palettes view
`<View><Colour>` - Colour palettes view
`<View><Gobo>` - Beam palettes view
`<View><Beam>` - Beam palettes view
`<View><Shape>` - Shape palettes view
`<View><FX>` - Shape palettes view

Also allows you to move/copy/delete workspace handles:
`<View><Move>`, `<View><Delete>` etc.